tests/data/formats/SPDXSBOMExample.tag

```diff
@@ -0,0 +1,72 @@
+# Document Information
+
+SPDXVersion: SPDX-2.2
+DataLicense: CC0-1.0
+DocumentNamespace: http://spdx.org/spdxdocs/spdx-document-xyz
+DocumentName: xyz-0.1.0
+SPDXID: SPDXRef-DOCUMENT
+
+
+# Creation Info
+
+Creator: Organization: Example Inc.
+Creator: Person: Thomas Steenbergen
+Created: 2020-07-23T18:30:22Z
+
+
+# Relationships
+
+Relationship: SPDXRef-Package-xyz CONTAINS SPDXRef-Package-curl
+
+
+# Relationships
```

Comment:
relationships header is duplicated

Reply:
I generated this example file with the cli convertor and didn't check the file in detail. So for this example file I will delete the duplicated header and also the superfluous header manually and open a new issue concerning the convertor since this behavior is not expected.

Reply:
done

Reply:
That sounds like a good solution, thanks!